tests/crm_test.py

```diff
@@ -1,3 +1,4 @@
+from importlib import reload
```

Comment:
Suggest to organize `from ...` imports _after_ `import ...` imports within the same group, i.e.:

```python
import os
import sys
from importlib import reload
```

Reply:
@jleveque updated